src/gnome-software.ui

```diff
@@ -452,6 +452,7 @@
                 <child>
```

Comment:
Commenting here because I cannot comment on the git message.
There's a small typo in _appearently_.